packages/dso-toolkit/src/components/label/label.scss

```diff
@@ -1,25 +1,35 @@
 @use "../label";
+@use "label.variables" as label-variables;
+@use "../../global/mixins/symbol.mixin" as symbol;
```

Comment:
`as symbol-mixins`, eigenlijk had de file ook zo moeten heten.

De conventie is `"component.mixins" as component-mixins";`. Alleen "standaard modules" (zonder `-suffix`) importeren we suffixloos.

Reply:
✅